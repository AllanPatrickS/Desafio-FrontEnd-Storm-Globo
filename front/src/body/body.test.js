import { cleanup } from '@testing-library/react';
import { create } from "react-test-renderer";
import Body from './body';

describe('Testing body', () => {

  it('renders correctly', () => {
    const wrapper = create(<Body />)
    expect(wrapper.toJSON()).toMatchSnapshot();
  });

  afterEach(function () {
    cleanup();
  });

});