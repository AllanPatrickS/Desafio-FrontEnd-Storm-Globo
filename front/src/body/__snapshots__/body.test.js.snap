// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing body renders correctly 1`] = `
<div
  className="MuiBox-root MuiBox-root-12 makeStyles-body-1"
>
  <div
    className="MuiContainer-root makeStyles-container-2"
  >
    <div
      className="MuiPaper-root MuiTableContainer-root makeStyles-tableContainer-3 MuiPaper-elevation1 MuiPaper-rounded"
    >
      <table
        aria-label="user table"
        className="MuiTable-root"
        role={null}
      >
        <thead
          className="MuiTableHead-root"
          role={null}
        >
          <tr
            className="MuiTableRow-root MuiTableRow-head"
            role={null}
          >
            <th
              aria-sort={null}
              className="MuiTableCell-root MuiTableCell-head makeStyles-headCell-5"
              scope="col"
            >
              <button
                className="MuiButtonBase-root MuiButton-root MuiButton-contained makeStyles-hiddenButton-8"
                disabled={false}
                onBlur={[Function]}
                onClick={[Function]}
                onDragLeave={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                style={
                  Object {
                    "borderRadius": 4,
                  }
                }
                tabIndex={0}
                type="button"
              >
                <span
                  className="MuiButton-label"
                >
                  Deletar
                </span>
              </button>
            </th>
            <th
              aria-sort={null}
              className="MuiTableCell-root MuiTableCell-head makeStyles-headCell-5"
              scope="col"
            >
              USUÁRIO
            </th>
            <th
              aria-sort={null}
              className="MuiTableCell-root MuiTableCell-head makeStyles-headCell-5"
              scope="col"
            >
              EMAIL
            </th>
            <th
              aria-sort={null}
              className="MuiTableCell-root MuiTableCell-head makeStyles-headCell-5"
              scope="col"
            >
              DATA DE INCLUSÃO
            </th>
            <th
              aria-sort={null}
              className="MuiTableCell-root MuiTableCell-head makeStyles-headCell-5"
              scope="col"
            >
              DATA DE ALTERAÇÃO
            </th>
            <th
              aria-sort={null}
              className="MuiTableCell-root MuiTableCell-head makeStyles-headCell-5"
              scope="col"
            >
              REGRAS
            </th>
            <th
              aria-sort={null}
              className="MuiTableCell-root MuiTableCell-head makeStyles-headCell-5"
              scope="col"
            >
              STATUS
            </th>
            <th
              aria-sort={null}
              className="MuiTableCell-root MuiTableCell-head makeStyles-headCell-5"
              scope="col"
            >
              AÇÕES
            </th>
          </tr>
        </thead>
        <tbody
          className="MuiTableBody-root"
          role={null}
        />
      </table>
    </div>
    <div
      aria-label="pagination"
      className="MuiButtonGroup-root makeStyles-pagination-6"
      role="group"
    >
      <button
        className="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButtonGroup-grouped MuiButtonGroup-groupedHorizontal MuiButtonGroup-groupedOutlined MuiButtonGroup-groupedOutlinedHorizontal MuiButtonGroup-groupedOutlined makeStyles-paginationButton-9 Mui-disabled Mui-disabled"
        disabled={true}
        onBlur={[Function]}
        onClick={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        style={
          Object {
            "borderRadius": 4,
          }
        }
        tabIndex={-1}
        type="button"
      >
        <span
          className="MuiButton-label"
        >
          Primeiro
        </span>
      </button>
      <button
        className="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButtonGroup-grouped MuiButtonGroup-groupedHorizontal MuiButtonGroup-groupedOutlined MuiButtonGroup-groupedOutlinedHorizontal MuiButtonGroup-groupedOutlined makeStyles-paginationButton-9 Mui-disabled Mui-disabled"
        disabled={true}
        onBlur={[Function]}
        onClick={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        style={
          Object {
            "borderRadius": 4,
          }
        }
        tabIndex={-1}
        type="button"
      >
        <span
          className="MuiButton-label"
        >
          Anterior
        </span>
      </button>
      <button
        className="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButtonGroup-grouped MuiButtonGroup-groupedHorizontal MuiButtonGroup-groupedOutlined MuiButtonGroup-groupedOutlinedHorizontal MuiButtonGroup-groupedOutlined makeStyles-page-10"
        disabled={false}
        onBlur={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        style={
          Object {
            "borderRadius": 4,
          }
        }
        tabIndex={0}
        type="button"
      >
        <span
          className="MuiButton-label"
        >
          0
        </span>
      </button>
      <button
        className="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButtonGroup-grouped MuiButtonGroup-groupedHorizontal MuiButtonGroup-groupedOutlined MuiButtonGroup-groupedOutlinedHorizontal MuiButtonGroup-groupedOutlined makeStyles-paginationButton-9 Mui-disabled Mui-disabled"
        disabled={true}
        onBlur={[Function]}
        onClick={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        style={
          Object {
            "borderRadius": 4,
          }
        }
        tabIndex={-1}
        type="button"
      >
        <span
          className="MuiButton-label"
        >
          Próximo
        </span>
      </button>
      <button
        className="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButtonGroup-grouped MuiButtonGroup-groupedHorizontal MuiButtonGroup-groupedOutlined MuiButtonGroup-groupedOutlinedHorizontal MuiButtonGroup-groupedOutlined makeStyles-lastButton-11 Mui-disabled Mui-disabled"
        disabled={true}
        onBlur={[Function]}
        onClick={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        style={
          Object {
            "borderRadius": 4,
          }
        }
        tabIndex={-1}
        type="button"
      >
        <span
          className="MuiButton-label"
        >
          último
        </span>
      </button>
    </div>
  </div>
</div>
`;
